filters/auth/oidc.go

```diff
@@ -698,7 +713,12 @@ func (f *tokenOidcFilter) Request(ctx filters.FilterContext) {
 			return
 		}
 		// 1. Client prepares an Authentication Request containing the desired request parameters.
-		f.doOauthRedirect(ctx)
+		// clear existing, invalid cookies
+		var purgeCookies = make([]*http.Cookie, len(cookies))
+		for i, c := range cookies {
+			purgeCookies[i] = deleteOidcCookie(c.Name, extractDomainFromHost(ctx.Request().Host, 1))
```

Comment:
`f.subdomainsToRemove` is not considered here

Reply:
Fixed that in https://github.com/zalando/skipper/pull/1965/commits/39c16f047e62edf7b25f8690565e2ca46ddda2f4